bundles/org.openhab.binding.metofficedatahub/src/main/java/org/openhab/binding/metofficedatahub/internal/dto/responses/SiteApiFeatureProperities.java

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
You seem to have a nice offset when typing this word 'properties' ;-) please adjust the filename.

Reply:
Thanks I don't know why I cant get that right- I put it down to my hearing - I will adjust any instances of that now to the correct spelling 👍